dotnet/Query/Query/TimestampComparer.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using Io.Cucumber.Messages.Types;
+
+namespace Io.Cucumber.Query
+{
+    // Port of io.cucumber.query.TimestampComparator (Java)
+    public class TimestampComparer : IComparer<Timestamp>
```

Comment:
Please pay attention to the access modifiers. This Java equivalent of this class is `package-private` (default, no access modifier). The .Net equivalent is `internal` I think.

Reply:
Also, this class is one I'm intending to move into messages.

Reply:
Modified the access modifiers where it makes sense to me. Let me know if you find any that remain concerning.
I left the Query.UpdateXXX methods as internal as some of them are used in the test project.